web-api/src/persistence/elasticsearch/searchClient.ts

```diff
@@ -6,6 +6,19 @@ import AWS from 'aws-sdk';
 
 const CHUNK_SIZE = 10000;
 
+export type SeachClientResultsType = {
```

Comment:
Are all of these types optional? Do we not know if aggregations, total, or results will exist?

Reply:
`Aggregations` is the only optional field here. Making the adjustment now.